src/MOF/schema/examples/quadratic.mof.json

```diff
@@ -0,0 +1,34 @@
+{
+    "description": "The problem: min{x^2 + x * y + y^2}",
```

Comment:
Based on 
```
{
     "coefficient": 2,
     "variable_1": "x",
     "variable_2": "y"
 }
```
it should be `min{x^2 + 2 * x * y + y^2}`, right?

Reply:
Oops. You're right that there is a mistake but that is the wrong one. I doubled the wrong value.

This is an annoying point that has come up a few times (e.g., https://github.com/JuliaOpt/LinQuadOptInterface.jl/issues/22).

MOI defines the [ScalarQuadraticFunction](http://www.juliaopt.org/MathOptInterface.jl/v0.1/apireference.html#MathOptInterface.ScalarQuadraticFunction) as `0.5 x' Q x`.